### PR TITLE
tuned: Prevent access to undefined variable

### PR DESCRIPTION
**Pull request by dperpeet**
Should replace #4225
Coverity found this.
Event though the code path is currently unused, it could
be used in the future.

### CODE REVIEW

**Comment by dperpeet** [comment]
I like using the check `!== undefined` here, because if something other than an event is passed here, the function isn't being used in the way it was intended to be.

File: pkg/tuned/dialog-view.jsx [hunk 1]
```diff
@@ -92,7 +92,8 @@ var DialogFooter = React.createClass({
                 if (self.props.dialog_done)
                     self.props.dialog_done(false);
             });
-        e.stopPropagation();
+        if (e !== undefined)
```
Comment (petervo) [inline pkg/tuned/dialog-view.jsx]: To make coverity happy this should check for null too.

**Comment by dperpeet** [comment]
I simplified the check to not just compare with `undefined` now, to also cover `null` values.

File: pkg/tuned/dialog-view.jsx (hunk 1 above)
Comment (dperpeet) [inline pkg/tuned/dialog-view.jsx]: > To make coverity happy this should check for null too.

Thanks, updated.

**Pull request closed without merging**